pattoo/api/portal/react/src/Modal.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+class Modal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick(event) {
+        this.props.modalClick(event);
+    }
+
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.